subfun/getData.m

```diff
@@ -2,9 +2,6 @@
 % The following structure will be the base that the function will use to run
 % the preprocessing pipeline according to the BIDS Structure
 
-% we check the option inputs and add any missing field with some defaults
-opt = checkOptions(opt);
```

Comment:
I would not remove this sub-function: it was created to make sure to add any missing fields in the `opt` structure that the user might have forgotten to specify

Reply:
That was probably a mistake from my side. I was just clearing spaces and probably removed it by mistake. 
The intended part is in the loop at the bottom of the script.